product_docs/docs/pgd/5/proxy/index.mdx

```diff
@@ -0,0 +1,119 @@
+
+PGD Proxy is a TCP layer 4 proxy. It can't encode/decode the PostgreSQL Wire Protocol. Although this may change in future releases.
+
+
+## How it works
+
+Upon starting, PGD Proxy connects to one of the endpoints given in the local config file and fetches DB connection information for all nodes, proxy options like listen address, listen port, routing details like current write leader, etc.
+
+The write leader election and routing is managed by BDR itself. Proxy internally maintains watchers and PubSub mechanism to get leader change notifications on Postgres notify/listen channels.
+
+All application client traffic then passes through the Proxy into the current write leader. If the write leader is not available then, Proxy disconnects all connection traffic until a new write leader is available. This also applies to circumstances when `pgd switchover` command is used to invoke a planned transition to a new write leader. The disconnect is immediate.
+
+## Installing PGD Proxy
+
+### Installing through TPAexec
+
+If PGD cluster is being deployed through TPAexec then it installs and configures PGD Proxy automatically. If you you wish to install PGD Proxy on any other node in PGD cluster then, you simply need to attach the `pgd-proxy` role to that instance in TPAexec's configuration file as shown below before deploying. See [TPAexec](/pgd/latest/deployments/tpaexec) for more information.
+```yaml
+- Name: proxy-a1
+  location: a
+  node: 4
+  role:
+  - pgd-proxy
+  volumes:
+  - device_name: /dev/sdf
+    volume_type: none
+```
+
+
+### Installing manually
+
+You can manually install the PGD Proxy on any Linux machine using `.deb` and `.rpm` packages available from the [BDR repository](https://techsupport.enterprisedb.com/software_subscriptions/add/products/bdr5/). The package name is `edb-pgd5-proxy`. For example:
+
+```sh
+# for Debian
+sudo apt-get install edb-pgd5-proxy
+```
+
+*Note:* You can install PGD Proxy on any BDR instance in the cluster, however it is not recommended.
+
+
+## Configuration
+
+Proxy connects to BDR database for its internal operations, like getting proxy configs, getting write leader details, etc. Therefore, it needs a list of endpoints/dsn to connect to BDR nodes. Proxy expects these configurations in a local config file `pgd-proxy-config.yml`. Following is a functional example of `pgd-proxy-config.yml` file:
+
+```yaml
+cluster:
+  name: my_pgd_cluster
+  endpoints:
+    - "host=bdr-a1 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a3 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a2 port=5432 dbname=bdrdb user=pgdproxy "
+  proxy:
+    name: "proxy-a1"
+```
+
+The `cluster.endpoints` and `cluster.proxy.name` are mandatory fields in the config file. Proxy always tries to connect to the first endpoint in the list, if it fails it tries the next endpoint and so on.
```

Comment:
I think we should mention somewhere about `bdr.node_routing_config_summary#route_dsn`...that is the endpoint proxy will use to connect